hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/io/storage/row/HoodieRowCreateHandle.java

```diff
@@ -272,11 +272,12 @@ private static Path makeNewPath(FileSystem fs, String partitionPath, String file
    *
    * @param partitionPath Partition path
    */
-  private static void createMarkerFile(String partitionPath,
+  private void createMarkerFile(String partitionPath,
                                        String dataFileName,
                                        String instantTime,
                                        HoodieTable<?, ?, ?, ?> table,
                                        HoodieWriteConfig writeConfig) {
+    stopIfAborted();
```

Comment:
Is creating marker file the right time to abort the tasks ?

Reply:
yes, it the task is already interrupted, we don't need to create marker, directly fail this task(at this time the file is not created yet, so it's safe here)